site/content/arangodb/3.11/graphs/enterprisegraphs/_index.md

```
@@ -33,16 +33,17 @@ This is known as the `smartGraphAttribute`, a value that is stored in every vert
-whenever possible. This approach provides significant advantages as it
-minimizes the impact of having suboptimal sharding keys defined when creating
-the graph.
-
-This means that, when using EnterpriseGraphs, the `smartGraphAttribute` is
-**not** required. As a consequence, you **cannot** define `_key` values on
-edges. 
+EnterpriseGraphs use hash-based sharding similar to General Graphs that
+pseudo-randomly distribute vertices across shards, but EnterpriseGraphs ensure that
+the adjacent edges of vertices are co-located in the same shard. As a consequence,
+you **cannot** define `_key` values for edges. You can also **not modify** the
+`_from ` and `_to` edge attributes because they contain the sharding key which
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)